vercel.json

```diff
@@ -1,12 +0,0 @@
-{
-  "version": 2,
-  "public": true,
-  "builds": [
-    {"src": "gifs/*.gif", "use": "@now/static"},
-    {"src": "package.json", "use": "@now/static-build"}
-  ],
-  "routes": [
-    {"handle": "filesystem"},
-    {"src": "/(.*)", "dest": "/gifs/$1.gif"}
```

Comment:
@styfle we need this or else e.g. https://g-git-fork-styfle-styfle-zero-config-matheus-projects-8231cede.vercel.app/coffin is 404, whilst https://g.vercel.app/coffin works

Reply:
Ah my bad. I missed that the suffix was removed from the path.

I guess this can't be done with zero config since removing the suffix will likely serve with the wrong content type 😔

Do you want to keep the other changes and I can add back vercel.json with a single rewrite?

Reply:
Added vercel.json in fcdfd3c